nemoguardrails/cli/__init__.py

```
@@ -196,7 +201,7 @@ def server(
     if default_config_id:
         api.set_default_config_id(default_config_id)  # Call function
 
-    uvicorn.run(server_app, port=port, log_level="info", host="0.0.0.0")
+    uvicorn.run(server_app, port=port, log_level="info", host="0.0.0.0", workers=workers)
```

Ruff (0.15.12)

[error] Possible binding to all interfaces

(S104)